frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -105,6 +109,8 @@ local symbol_prefix = {
         mem_usage = C_("FooterCompactItemsPrefix", "M"),
         wifi_status = "",
         wifi_status_off = "",
+        page_turning_inverted = "PgT:I",
+        page_turning_regular = "PgT:R",
```

Comment:
No matter how you like your "PgT", I don't think this can be the 4 letters among the "compact" where everything else is a single glyph.